packages/@glimmer/syntax/tests/parser-node-test.ts

```diff
@@ -474,11 +474,23 @@ test("allow {{undefined}} to be passed as a param", function() {
 test("Handlebars partial should error", function(assert) {
   assert.throws(() => {
     parse("{{> foo}}");
-  }, Error(`Handlebars partials are not supported: "{{> foo" on line "1:0"`));
+  }, Error(`Handlebars partials are not supported: "{{> foo" at L1:C0`));
 });
 
 test("Handlebars partial block should error", function(assert) {
   assert.throws(() => {
     parse("{{#> foo}}{{/foo}}");
-  }, new Error(`Handlebars partial blocks are not supported: "{{#> foo" on line "1:0"`));
+  }, new Error(`Handlebars partial blocks are not supported: "{{#> foo" at L1:C0`));
+});
+
+test("Handlebars decorator should error", function(assert) {
+  assert.throws(() => {
+    parse("{{* foo}}");
+  }, new Error(`Handlebars decorators are not supported: "{{* foo" at L1:C0`));
+});
+
+test("Handlebars decorator block should error", function(assert) {
+  assert.throws(() => {
+    parse("{{#* foo}}{{/foo}}");
+  }, new Error(`Handlebars decorator blocks are not supported: "{{#* foo}}{{/foo}}" at L1:C0`);
```

Comment:
Updated to match partials, let me know if I'm missing anyway, otherwise seems good to go.